src/app/public/src/modules/nav/nav.component.spec.ts

```diff
@@ -101,6 +101,15 @@ describe('StacheNavComponent', () => {
     expect(route.isCurrent).toBe(true);
   });
 
+  it('should return true if a given route is external', () => {
+    component.routes = [{ name: 'Test', path: 'https://google.com' }];
+    const route = component.routes[0];
+    let result = component['isExternal'](route);
```

Comment:
So, the reason coverage is failing is because you're calling the private method explicitly, instead of allowing it to be called implicitly via the public `navigate()` method. Just need to rework the unit test.